system/Autoloader/Autoloader.php

```diff
@@ -280,6 +280,15 @@ protected function loadInNamespace(string $class)
 	{
 		if (strpos($class, '\\') === false)
 		{
+			$class    = 'Config\\' . $class;
+			$filePath = APPPATH . str_replace('\\', '/', $class) . '.php';
```

Comment:
Use `DIRECTORY_SEPARATOR` as replacement for pretty printing shutdown handler backtrace.

Reply:
updated.